dev-infrastructure/svc-pipeline.yaml

```diff
@@ -151,8 +153,10 @@ resourceGroups:
       configRef: svc.istio.versions
     dependsOn:
     - prometheus
-  # - updates workload to use istio on version svc.istio.targetVersion
-  # - configures istio IP tag usage
+    shellIdentity:
+      configRef: aroDevopsMsiId
+      # - updates workload to use istio on version svc.istio.targetVersion
+      # - configures istio IP tag usage
```

Comment:
annoyingly, `make yamlfmt` thinks these comments are for the `configRef`, not the next step ... another reason not to have commetns :)